### ISSUE TITLE
Wrong traffic light screenReaderText when score is "na"

### ISSUE THREAD

**Issue by afercia** [Yoast/YoastSEO.js#433]
Looks like when the score is "na", the returned screen reader text is "Bad SEO score" while it should say that the focus keyword is missing. To reproduce:
- remove the focus keyword
- the traffic lights turn all to grey
- check the SVG alt attribute, it says "Bad SEO score"

See also https://github.com/Yoast/wordpress-seo/pull/4297
